src/Build/Logging/FileLogger.cs

```diff
@@ -81,6 +81,11 @@ private void InitializeFileLogger(IEventSource eventSource, int nodeCount)
             // want to make decisions based on our verbosity, so we do this last.
             base.Initialize(eventSource, nodeCount);
 
+            if (!SkipProjectStartedText && Verbosity >= LoggerVerbosity.Normal)
+            {
+                eventSource.BuildStarted += (obj, args) => WriteHandler($"Verbosity is set to: {Verbosity}.");
```

Comment:
This string needs to be localized. Can you add a new string to [`src/Build/Resources/Strings.resx`](https://github.com/microsoft/msbuild/blob/master/src/Build/Resources/Strings.resx) and use it here with `ResourceUtilities.FormatResourceStringIgnoreCodeAndKeyword()`?

Reply:
Yes, sure - I appreciate your help as always, thanks 😃 
I should have known that localization is required 🙄